### ISSUE TITLE
distance training models

### ISSUE THREAD

**Issue by Hogye** [HassanBinHaroon/YOLOER_V2#1]
Hello, can you provide some relevant distance training models?

**Comment by HassanBinHaroon** [comment]
![Screenshot 2023-03-27 135458](https://user-images.githubusercontent.com/104098902/227892800-7c08517f-84de-4cf9-9912-f3c3c2c0cc62.png)

This is one of the model in the present repository.

**Comment by Hogye** [comment]
Yes, I want to know the structure of this model. Can you share the code of this model? And how can we get information about this distance? What is the principle?

**Comment by HassanBinHaroon** [comment]
> Yes, I want to know the structure of this model. Can you share the code of this model? And how can we get information about this distance? What is the principle?

Its a simple neural network now. 

This project is in progress and the distance estimation model is not accurate at the moment. 

Thanks!
